# v1.0 , last modified 22/02/2022
# message all float types a0,a2,a3,a4,t0,tf 
# added by Kumar Sindhurakshit on 22/02/2022

float64 a0
float64 a1
float64 a2
float64 a3
float64 t0
float64 tf

